### ISSUE TITLE
Ability to create patient record on the server from an existing patient record

### ISSUE THREAD

**Issue by martinburchell** [ucam-department-of-psychiatry/camcops#300]
Scenario: 

On admission to ward, inpatient fills in questionnaires (using clinician mode) but doesn't manage to repeat the questionnaires before being discharged. We need to create another record for this patient on the server so they can do the questionnaires at home (using single user mode). It would be nice to be able to copy the patient's personal information from the existing patient record to this new server record. 

Currently we have to copy and paste. We'd need to restrict this to patients in the user's group.

Nice-to-have, not urgently required.

**Comment by RudolfCardinal** [comment]
Yes, good thought. May be worth thinking this through in the context of a broader central patient registration system, to reduce effort and provide accuracy constraints (e.g. the capability to take a feed from the main EPR for demographics and potentially the ability to lock users to that -- there is already the hook ``op_validate_patients``).
